src/diffusers/pipelines/wuerstchen/pipeline_wuerstchen.py

```diff
@@ -390,6 +361,9 @@ def __call__(
         latents = self.vqgan.config.scale_factor * latents
         images = self.vqgan.decode(latents).sample.clamp(0, 1)
 
+        # Offload all models
+        self.maybe_free_model_hooks()
```

Comment:
@patrickvonplaten same as above. Some KT would be nice.

Reply:
Sorry don't fully understand here what do you mean by KT?

Reply:
Leftover from https://github.com/huggingface/diffusers/pull/4514

Reply:
Knowledge transfer.